include/singa/log.h

```diff
@@ -0,0 +1,29 @@
+/************************************************************
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+*************************************************************/
+
+#ifndef SINGA_LOG_H_
+#define SINGA_LOG_H_
+
+namespace singa {
+    extern bool enable_info_log;
```

Comment:
Will replace it with verbose_singa.

If its declaration is put in singa.h, then every src file where this var appears should include singa.h as well as the files included in singa.h. Is it a bit redundant? 

Since this is a global variable and defined as external, its real declaration and initialization should be put somewhere else.  Which file do you think is appropriate for it?

btw, where is the main entrance(function) for singa located, singa/main.cc or utils/tool.cc?